scripts/script.test.js

```diff
@@ -0,0 +1,27 @@
+import { translate } from "./translator";
+
+describe("Test cases for a function that automatically detects a string and converts it to or from morse code and latin text", () => {
+  it("should correctly convert English text to morse code", () => {
```

Comment:
Make smaller tests, characters, numbers, non-alphanumeric characters, words, single chars stuff like that